spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -1,52 +1,52 @@
 describe MiqExpression do
   describe "#to_sql" do
     it "generates the SQL for an EQUAL expression" do
-      sql, * = MiqExpression.new("EQUAL" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name = 'foo'")
+      sql, * = MiqExpression.new("EQUAL" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) = 'foo'")
     end
 
     it "generates the SQL for a = expression" do
-      sql, * = MiqExpression.new("=" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name = 'foo'")
+      sql, * = MiqExpression.new("=" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) = 'foo'")
     end
 
     it "generates the SQL for a LIKE expression" do
-      sql, * = MiqExpression.new("LIKE" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name LIKE '%foo%'")
+      sql, * = MiqExpression.new("LIKE" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) LIKE '%foo%'")
```

Comment:
a) I've added ILIKE to arel. Haven't hooked it up to active record. to be honest, active record uses ilike not like.

I seem to remember for some locales and string, `x.downcase` != `LOWER(x). Or maybe that is `upcase`.